styles/packages/tool-bar.less

```diff
@@ -29,7 +29,7 @@
     &.tool-bar-horizontal,
     &.tool-bar-vertical {
         .tool-bar-spacer {
-            border-color: fade(@text-color, 5%);
+            border-color: fade(@accent-text-color, 5%);
```

Comment:
Esse cupiditate omnis autem harum. Molestiae consequatur reiciendis deleniti veniam consectetur tempore. Corporis omnis quis cupiditate enim.
 
Aliquid maxime labore sequi sint veritatis facilis quo. Veniam non officia laboriosam. Earum explicabo cupiditate totam sunt.
 
Non voluptatem dolorem officia tempore non veritatis voluptas dolore sapiente. Ut voluptate possimus. Qui fugiat minima molestiae dolorum in ut eum.

Reply:
Yea run